src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.Linux.cs

```diff
@@ -76,32 +76,7 @@ internal static DateTime BootTimeToDateTime(TimeSpan timespanAfterBoot)
         }
 
         /// <summary>Gets the system boot time.</summary>
-        private static DateTime BootTime
-        {
-            get
-            {
-                // '/proc/stat -> btime' gets the boot time.
-                // btime is the time of system boot in seconds since the Unix epoch.
-                // It includes suspended time and is updated based on the system time (settimeofday).
-                const string StatFile = Interop.procfs.ProcStatFilePath;
-                string text = File.ReadAllText(StatFile);
-                int btimeLineStart = text.IndexOf("\nbtime ", StringComparison.Ordinal);
-                if (btimeLineStart >= 0)
-                {
-                    int btimeStart = btimeLineStart + "\nbtime ".Length;
-                    int btimeEnd = text.IndexOf('\n', btimeStart);
-                    if (btimeEnd > btimeStart)
-                    {
-                        if (long.TryParse(text.AsSpan(btimeStart, btimeEnd - btimeStart), out long bootTimeSeconds))
-                        {
-                            return DateTime.UnixEpoch + TimeSpan.FromSeconds(bootTimeSeconds);
-                        }
-                    }
-                }
-
-                return DateTime.UtcNow;
-            }
-        }
+        private static DateTime BootTime => DateTimeOffset.FromUnixTimeMilliseconds(((long)Interop.Sys.GetTimestamp() / 1_000_000)).DateTime;
```

Comment:
I still don't understand.  Can you please explain how this is computing the boot time of the machine?